### PR TITLE
[1LP][RFR] Renamed some tests to avoid duplicates

### PR DESCRIPTION
**Pull request by psav**
Renamed some tests to avoid duplicates

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 67ddd1e8670b8806270079f6bef21e79bd2b1e36:

`cfme/tests/cloud/test_provisioning.py`:
- :bangbang: Line 488:10: **E126** *continuation line over-indented for hanging indent*

`cfme/tests/infrastructure/test_advanced_search_host.py`:
- :bangbang: Line 104:101: **E501** *line too long (105 > 100 characters)*
- :bangbang: Line 225:101: **E501** *line too long (102 > 100 characters)*

`cfme/tests/infrastructure/test_advanced_search_providers.py`:
- :bangbang: Line 196:101: **E501** *line too long (101 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/infrastructure/test_advanced_search_providers.py [hunk 1]
```diff
@@ -44,112 +43,118 @@ def advanced_search_view():
...
 
-def test_filter_save_and_cancel_load(rails_delete_filter, advanced_search_view):
+def test_provider_filter_save_and_cancel_load(rails_delete_filter, advanced_search_view):
     # bind filter_name to the function for fixture cleanup
-    test_filter_save_and_cancel_load.filter_name = fauxfactory.gen_alphanumeric()
-    logger.debug('Set filter_name to: {}'.format(test_filter_save_and_cancel_load.filter_name))
+    test_provider_filter_save_and_cancel_load.filter_name = fauxfactory.gen_alphanumeric()
+    logger.debug('Set filter_name to: {}'.format(
+        test_provider_filter_save_and_cancel_load.filter_name)
+    )
```
Comment (jamesooden) [inline cfme/tests/infrastructure/test_advanced_search_providers.py]: You know I much prefer putting the closing parens down here at the indentation level of the call (i.e. I like that you do that.   There is so much code that does the opposite:
   
`some_func(
    blah,
    blather)'

That I thought the way you were doing it was not our standard.

File: cfme/tests/services/test_request.py [hunk 2]
```diff
@@ -18,7 +17,7 @@
 ]
 
 
-def test_copy_request(appliance, setup_provider, provider, catalog_item, request):
+def test_copy_request_bz1194479(appliance, setup_provider, provider, catalog_item, request):
```
Comment (jamesooden) [inline cfme/tests/services/test_request.py]: I wonder if this naming convention when automating tests for BZ's should be part of our standard (i.e. test_$blah_bz$bznum)

**Comment by jamesooden** [comment]
If you get rid of the lint problems and PRT is clean I'll approve this.   It all looks good.   BTW, this is a perfect example of something you could talk to all of us about, which is naming our tests more uniquely.

**Comment by RonnyPfannschmidt** [comment]
we might want to include a actual warning that tells us about already used test names in future

**Comment by dajoRH** [comment]
Lint report for commit 2a4de5902cf3d089c5fdd41764e33690b3d389ed:

`cfme/tests/cloud/test_provisioning.py`:
- :bangbang: Line 490:10: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cc89f1eabf70540487553d05ba5c937f9ff8c25c:

`cfme/tests/services/test_orchestration_template.py`:
- :red_circle: Line 8:1: **F401** *'cfme.cloud.provider.CloudProvider' imported but unused*
- :red_circle: Line 10:1: **F811** *redefinition of unused 'update' from line 6*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/configure/test_tag.py [hunk 3]
```diff
@@ -115,7 +115,7 @@ def vm(self, request, a_provider, appliance):
         return _vm(request, a_provider, appliance.rest_api)
 
     @pytest.mark.tier(2)
-    def test_edit_tags(self, appliance, tags):
+    def test_edit_tags_rest(self, appliance, tags):
```
Comment (mkoura) [inline cfme/tests/configure/test_tag.py]: OPTIONAL: the test is in the ``TestTagsViaREST``class meaning the current complete testcase name is ``TestTagsViaREST.test_edit_tags``. But I can live with that redundant ``_rest`` suffix.

**Pull request merged**